clear-env.sh

```diff
@@ -1,25 +1,35 @@
 #!/bin/bash
 
-# usage:
+# Usage
 #   ./clear-env.sh $ENV
```

Comment:
It would be nice to have a small description of what the script does too.

Reply:
Updated!